crates/bevy_audio/src/audio_output.rs

```diff
@@ -81,15 +113,28 @@ pub(crate) fn play_queued_audio_system<Source: Asset + Decodable>(
         return;
     };
 
-    for (entity, source_handle, settings, spatial) in &query_nonplaying {
+    for (entity, source_handle, settings, maybe_emitter_transform) in &query_nonplaying {
         if let Some(audio_source) = audio_sources.get(source_handle) {
             // audio data is available (has loaded), begin playback and insert sink component
-            if let Some(spatial) = spatial {
+            if settings.spatial {
+                let (left_ear, right_ear) = listener.get_ear_positions();
+
+                if listener.query.iter().len() > 1 {
+                    warn!("Multiple SpatialListeners found. Using first.");
```

Comment:
"first" is not very helpful. Could you log the entity used?

Reply:
Good idea!